orienteer-core/src/main/java/org/orienteer/core/component/widget/OModuleManagerWidget.java

```diff
@@ -0,0 +1,61 @@
+package org.orienteer.core.component.widget;
+
+import com.google.inject.Inject;
+import com.google.inject.Injector;
+import com.orientechnologies.orient.core.record.impl.ODocument;
+import org.apache.wicket.markup.html.link.Link;
+import org.apache.wicket.model.IModel;
+import org.apache.wicket.model.Model;
+import org.orienteer.core.component.FAIcon;
+import org.orienteer.core.component.FAIconType;
+import org.orienteer.core.service.Reload;
+import org.orienteer.core.service.ReloadFilter;
+import org.orienteer.core.widget.AbstractWidget;
+import org.orienteer.core.widget.Widget;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+/**
+ * @author Vitaliy Gonchar
+ */
+@Widget(domain="schema", tab="classes", id="load-module", autoEnable=true)
+public class OModuleManagerWidget extends AbstractWidget<Void> {
+
+    private static final Logger LOG = LoggerFactory.getLogger(OModuleManagerWidget.class);
+
+    @Inject
+    private Injector injector;
+
+    public OModuleManagerWidget(String id, IModel<Void> model, IModel<ODocument> widgetDocumentModel) {
+        super(id, model, widgetDocumentModel);
+        add(new Link<Void>("reload") {
+            @Override
+            public void onClick() {
+                LOG.debug("Click on reload");
+                reload();
+                setResponsePage(new ReloadPage());
+            }
+
+            private void reload() {
+                ReloadFilter reloadFilter = injector.getInstance(ReloadFilter.class);
+                Reload reload = new Reload(reloadFilter);
```

Comment:
Must be small delay for send to user reload page. And user does not see page with 500 error.
May be this variant better:
 final ScheduledThreadPoolExecutor executor = new ScheduledThreadPoolExecutor(5);
 executor.schedule(new Reload(orienteerFilter), 3, TimeUnit.SECONDS);